packages/noports_core/lib/src/sshnp/mixins/sshnp_local_file_handler.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:noports_core/src/sshnp/sshnp_impl.dart';
+import 'package:noports_core/src/sshnp/sshnp_result.dart';
+import 'package:noports_core/utils.dart';
+
+mixin SSHNPLocalFileHandler on SSHNPImpl {
+  final LocalSSHKeyUtil _sshKeyUtil = LocalSSHKeyUtil();
+  @override
+  LocalSSHKeyUtil get keyUtil => _sshKeyUtil;
+
```

Comment:
A mixin on SSHNPImpl which makes the local file system safely available. At init time, it checks whether the platform is valid to run the local filesystem (i.e. mac/linux/windows). Also exposes the LocalSSHKeyUtil